### ISSUE TITLE
Framework Nuclei SDK version is not updated

### ISSUE THREAD

**Issue by fanghuaqi** [Nuclei-Software/platform-nuclei#8]
This framework used by platform-nuclei is not up to date, request to update it, current latest version of nuclei-sdk is 0.2.9, but the one used in platform-nuclei is 0.2.6.

**Comment by fanghuaqi** [comment]
@ivankravets  Is there a way for user to experiment with nuclei-sdk daily version directly from github: https://gitee.com/Nuclei-Software/nuclei-sdk

Thanks
Huaqi

**Comment by fanghuaqi** [comment]
Released version of nuclei-sdk could be found here https://github.com/Nuclei-Software/nuclei-sdk/releases

**Comment by ivankravets** [comment]
Do you mean https://docs.platformio.org/en/latest/projectconf/section_env_platform.html#platform-packages ?

**Comment by ivankravets** [comment]
Also, there is an example https://docs.platformio.org/en/latest/platforms/espressif32.html#using-arduino-framework-with-staging-version

**Comment by fanghuaqi** [comment]
> 
> 
> Also, there is an example https://docs.platformio.org/en/latest/platforms/espressif32.html#using-arduino-framework-with-staging-version

Yeah, I think this could solve my question.

**Comment by fanghuaqi** [comment]
Hi @ivankravets , I have cut `0.3.0` release of Nuclei SDK, see _https://github.com/Nuclei-Software/nuclei-sdk/releases/tag/0.3.0_, and I use `pio package publish` to publish the package, but it seems that it is not yet approved, could you help check it.

~~~
(base) D:\workspace\Sourcecode\nuclei-sdk>pio package publish c:\Users\57856\Downloads\framework-nuclei-sdk-0.3.0.tar.gz
The package has been accepted for moderation. We will notify you on email when process it.
~~~

Thanks
Huaqi

**Comment by ivankravets** [comment]
Hi,

No, the package will not be approved. See 

- https://github.com/Nuclei-Software/platform-nuclei/blob/master/platform.json#L39
- https://docs.platformio.org/en/latest/core/userguide/package/cmd_publish.html#cmdoption-pio-package-publish-owner

Please re-publish.

**Comment by fanghuaqi** [comment]
Republish the package use `pio package publish --owner nuclei c:\Users\57856\Downloads\framework-nuclei-sdk-0.3.0.tar.gz`,

BTW, @ivankravets I want to ask it is a web ui interface or via PlatformIO IDE where I can upload package, and where can I see the account or organization information of platformio in the website, I tried to find it, but I can't.

Thanks
Huaqi

**Comment by ivankravets** [comment]
Thanks! Should be approved now.

Happy Holidays!

**Comment by ivankravets** [comment]
Please repeat the same `pio package publish --owner nuclei` for the recently released dev-platform 1.0.11.

**Comment by fanghuaqi** [comment]
Hi @ivankravets, I have uploaded `platform-nuclei 1.0.11`, and found some issue when debug in IDE, and fire a issue, see https://github.com/platformio/platformio-vscode-ide/issues/2280
